charts/datadog/templates/daemonset.yaml

```
@@ -58,6 +58,9 @@ spec:
         container.seccomp.security.alpha.kubernetes.io/system-probe: {{ .Values.datadog.systemProbe.seccomp }}
         {{- end }}
         {{- end }}
+        {{- if and .Values.datadog.sbom.containerImage.uncompressedLayersSupport }}
```

we can add it only if appArmor is enabled too

```suggestion
        {{- if and .Values.agents.podSecurity.apparmor.enabled .Values.datadog.sbom.containerImage.uncompressedLayersSupport }}
```

---

It was therefore - hence the "and" - but I probably removed it by mistake